src/recording/macromanager.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include <QDebug>
+#include <QObject>
+#include <QtCore>
+
+#include "engine/enginemaster.h"
+#include "mixer/playermanager.h"
+#include "preferences/usersettings.h"
+#include "recording/recordingmanagerbase.h"
+
+#define MACRORECORDING_PREF_KEY "[MacroRecording]"
```

Comment:
Use a constexpr

Reply:
It seems I have to use const, since it's a string.